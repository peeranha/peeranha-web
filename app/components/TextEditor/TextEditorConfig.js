const TextEditorConfig = {
  apiKey: 'fvry0is6hvuc3ij671v897r0t3854qvo6ne0k5eun4378zxk',
  plugins: [
    'codesample textpattern advlist autolink lists link image charmap print preview anchor textcolor',
    'searchreplace visualblocks code fullscreen',
    'insertdatetime media table contextmenu paste code help wordcount',
  ],
  toolbar:
    'insert | undo redo |  formatselect | bold italic backcolor  | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | removeformat | help',
};

export default TextEditorConfig;
