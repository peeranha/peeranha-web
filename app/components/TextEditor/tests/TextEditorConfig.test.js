import TextEditorConfig from '../TextEditorConfig';

describe('TextEditorConfig', () => {
  it('snapshot test', () => {
    expect(TextEditorConfig).toMatchSnapshot();
  });
});
