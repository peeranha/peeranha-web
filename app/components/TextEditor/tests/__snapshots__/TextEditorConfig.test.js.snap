// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextEditorConfig snapshot test 1`] = `
Object {
  "apiKey": "fvry0is6hvuc3ij671v897r0t3854qvo6ne0k5eun4378zxk",
  "plugins": Array [
    "codesample textpattern advlist autolink lists link image charmap print preview anchor textcolor",
    "searchreplace visualblocks code fullscreen",
    "insertdatetime media table contextmenu paste code help wordcount",
  ],
  "toolbar": "insert | undo redo |  formatselect | bold italic backcolor  | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | removeformat | help",
}
`;
